docs/components/Splash.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from "react";
+import styles from "./Splash.module.css";
+
+const Splash = ({ theme }) => {
+  const [isSplashOpen, setSplashOpen] = useState(false);
+  const [display, setDisplay] = useState(false);
```

Comment:
not blocking since we're in a rush prepping for eth global, but as a general note, we should lean on things like [Radix's dialog](https://www.radix-ui.com/docs/primitives/components/dialog) for accessibility, otherwise we're missing out on support for things like keyboard/tab navigation, screen readers, etc.

Reply:
Noted, but this should all get deleted post-hackathon so should be fine for now